lib/auth/permissions.go

```diff
@@ -612,8 +614,9 @@ func definitionForBuiltinRole(clusterName string, recConfig types.SessionRecordi
 			role.String(),
 			types.RoleSpecV6{
 				Allow: types.RoleConditions{
-					Namespaces:     []string{types.Wildcard},
-					DatabaseLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
+					Namespaces:            []string{types.Wildcard},
+					DatabaseLabels:        types.Labels{types.Wildcard: []string{types.Wildcard}},
+					DatabaseServiceLabels: types.Labels{types.Wildcard: []string{types.Wildcard}},
```

Comment:
What does database service need this for? Does it every query other database services?

Reply:
It shouldn't :+1: 
Removed